mozartdata/transforms/goodr_reporting/goodr_com_usage_kpis.sql

```diff
@@ -0,0 +1,198 @@
+        total_customers:
+            This is the total number of users that bought a product in a day.
+    Todos:
+        - handle leap years
+    More to come
+
+ */
+with
+    shopify_data_this_ytd as (
+                                 select
+                                     d.date::date                               as date
+                                   , shopify.sessions
+                                   , shopify."ONLINE STORE VISITORS"            as users
+                                   , shopify."NEW CUSTOMERS"                    as new_customers
+                                   , shopify.customers                          as total_customers
+                                   , shopify."SESSIONS THAT COMPLETED CHECKOUT" as sess_completed_checkout
+                                   , shopify."CONVERSION RATE"                  as daily_conversion_rate
+                                 from
+                                     dim.date                                       as d
+                                     inner join
+                                         shopify_exports.kpi_data_20240101_20250206 as shopify
+                                             on
+                                             d.date = shopify.day::date
+                                 where
+                                       d.date >= date_trunc('year', current_date)
+                                   and d.date <= current_date
+                             )
+  , shopify_data_last_ytd as (
+                                 select
+                                     d.date::date                               as date
+                                   , shopify.sessions
+                                   , shopify."ONLINE STORE VISITORS"            as users
+                                   , shopify."NEW CUSTOMERS"                    as new_customers
+                                   , shopify.customers                          as total_customers
+                                   , shopify."SESSIONS THAT COMPLETED CHECKOUT" as sess_completed_checkout
+                                   , shopify."CONVERSION RATE"                  as daily_conversion_rate
+                                 from
+                                     dim.date                                       as d
+                                     inner join
+                                         shopify_exports.kpi_data_20240101_20250206 as shopify
+                                             on
+                                             d.date = shopify.day::date
+                                 where
+                                       d.date >= date_trunc('year', dateadd(year, -1, current_date))
+                                   and d.date <= dateadd(year, -1, current_date)
+                             )
+  , media_totals_this_ytd as (
+                                 select
+                                     date
```

Comment:
yeah... I wish we had renamed this column. but it's kind of too late now with how much that report is used. Sorry I didn't catch it before. 

But let's fully qualify that, since date is reserved.
```suggestion
                                     pm.date
```

Reply:
100, can't believe I missed that